pmg/views.py

```diff
@@ -226,24 +226,25 @@ def get_year_unicode(m_date):
     recent_questions = load_from_api(
         'minister-questions-combined',
         params=params)['results']
+
     all_meetings = [m for m in committee['events'] if m['type'] == 'committee-meeting' and get_year_unicode(m['date']) >= YEAR_CUTOFF]
 
     for meeting in all_meetings:
         meeting_year = get_year_unicode(meeting['date'])
 
-        if meeting_year not in meetings_by_year:
-            meetings_by_year[meeting_year] = []
+        if meeting_year not in filtered_meetings:
+            filtered_meetings[meeting_year] = []
+
+        filtered_meetings[meeting_year].append(meeting)
 
-        meetings_by_year[meeting_year].append(meeting)
+    filtered_meetings['six-months'] = [m for m in all_meetings if (now.month - get_month_unicode(m['date']) <= 6) and (get_year_unicode(m['date']) == now.year)]
```

Comment:
This ain't gonna work for the first few months of the year (it'll ignore everything in the previous year).

Perhaps [parse the event date](http://stackoverflow.com/questions/969285/how-do-i-translate-a-iso-8601-datetime-string-into-a-python-datetime-object) and the just use ``(now - event_data).days < 30*6``?